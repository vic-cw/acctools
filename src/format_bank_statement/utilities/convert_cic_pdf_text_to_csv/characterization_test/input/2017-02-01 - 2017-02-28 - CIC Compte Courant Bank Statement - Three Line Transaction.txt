                 Crédit Industriel et Commercial

                                                       CIC PARIS SAINT  MARTIN

                                             224 RUE SAINT MARTIN 75010 PARIS
            0820 82 03 14 (Service 0,12 /min + prix appel) FAX 01 44 16 81 39
                                                                                  20314@cic.fr  BIC : CMCIFRPP

                                                                                                                                                            51155

                                                                                                                                                            21566

                                                               M  JOHN  DOE

                                                               4  BROADWAY     AVENUE

                                                               75001      PARIS

...........................................................................................................................V..O...T.R..E...C..O..N..S..E..I.L..L..E..R..:..M...S..é..b..a.s..t.i.e.n...D..U..P..O..N.T.....................

RELEVE ET INFORMATIONS BANCAIRES                                                                        Agence 20314

28 février 2017

      C/C CONTRAT PERSONNEL GLOBAL N° 01234567890 en euros (GD)

      TITULAIRE(S) : M JOHN DOE

Date        Date valeur   Opération                                                      Débit euros    Crédit euros

                          SOLDE CREDITEUR AU 31/01/2017                                                                                                     1.234,56                                                                                                                                         Pour tout appel en vue d'obtenir la bonne exécution d'un contrat ou le traitement

02/02/2017  02/02/2017    CHEQUE 9876543 T012-3456-78910                                        52,50

02/02/2017  02/02/2017    VIR ULYSSES TECHNOLOGIES (PT)                                                                                                      33,94                                                                             d'une réclamation, appelez le 09 69 32 06 06 appel non surtaxé

02/02/2017  02/02/2017    VIR ULYSSES TECHNOLOGIES (PT)                                                                                                      591,31

06/02/2017  06/02/2017    VIR SEPA P1234567                                              5.001,00

06/02/2017  06/02/2017    PAIEMENT CB      0204 MONTREUIL                                          66,91

                          NATURALIA        CARTE 00123456

08/02/2017  08/02/2017    PRLV SEPA I. LIKE. TO. WALK                                    935,00

                          ABC : AB12YYY001234

                          DEF : GGAB12YYY0012345678910111213F1415161

08/02/2017  08/02/2017    PAIEMENT CB      0701 BAGNOLET                                   73,00

                          ABCD             CARTE 00123456

10/02/2017  01/02/2017    F RETRO CP.GLOBAL                                                                                                                 2,30

27/12/2016  13/02/2017    EXPENSE ON TWO LINES                                              20,00

(GE) : protégé par la Garantie de l'Etat                                                                <<Suite au verso>>

(GD) : protégé par la Garantie des Dépôts

www.garantiedesdepots.fr

                                                                                                                                                            Page 1

  CREDIT INDUSTRIEL ET COMMERCIAL (CIC) - Banque régie par les articles L.511-1 et suivants du Code Monétaire et Financier - SA au capital de 608 439 888 

6, avenue de Provence 75009 Paris - Adresse postale : 75452 Paris cedex 09 - tél 01 45 96 96 96 - swift CMCIFRPP - www.cic.fr - 542 016 381 RCS Paris - TVA intracommunautaire : FR34542016381

M le Médiateur du CIC : 63 chemin Antoine Pardon 69160 Tassin la demi lune - Pour les opérations effectuées en sa qualité d'intermédiaire en opérations d'assurance : ORIAS 07 025 723 (www.orias.fr)
RELEVE ET INFORMATIONS BANCAIRES                                                                  Agence 20314

28 février 2017

      C/C CONTRAT PERSONNEL GLOBAL N° 01234567890                en   euros (GD)

      TITULAIRE(S) : M JOHN DOE

Date        Date valeur       Opération                                           Débit euros     Crédit euros

                          LINE TWO

02/01/2017  13/02/2017    PROUT PROUT                                             8,00

14/02/2017  14/02/2017    PAIEMENT CB      1234 MONTREUIL                         10,66

                          PHARMACIE ABCDE        CARTE 00123456

24/02/2017  24/02/2017    INCOMING FUNDS                                                          1.234,56

                          FUNDS LINE 2          /J01C23456789101

Réf : 002                     SOLDE CREDITEUR AU 28/02/2017                                        1.234,56

QXBAN : AB12 CMCI FRPP XXX CDE3 FGHI 4JKL MNOP 5QR                    IBAN : FR76 3000 1234 5678  9101 1121 314

SITUATION DE VOS CREDITS en euros au 28 février 2017

Numéro                        Libellé du compte                            Montant emprunté       Solde (*)

00012345678      ALLURE LIBRE                                         EUR         +0,00           +0,00

00012345679      ETALIS                                               EUR         +0,00           +0,00

(*) Solde correspondant à l'amortissement normal du prêt

AUTORISATIONS DE DECOUVERTS

N° de compte                  Libellé                                                             Montant euros

00012345678      DECOUVERT C. PERS. GLOBAL TAEG 15,12% AU 27/01/2017                              500.000,00

(GE) : protégé par la Garantie de l'Etat

(GD) : protégé par la Garantie des Dépôts

www.garantiedesdepots.fr

                                                                                                  Page 2
                 Crédit Industriel et Commercial

                                              CIC PARIS SAINT MARTIN

                                           224 RUE SAINT MARTIN 75010 PARIS
  0820 82 03 14 (Service 0,12 /min + prix appel) FAX 01 44 16 81 39
                                                                           20314@cic.fr  BIC : CMCIFRPP

RELEVE ET INFORMATIONS BANCAIRES                                                         Agence 20314

28 février 2017

TOTAL MENSUEL DES FRAIS DEBITES SUR PRODUITS ET SERVICES BANCAIRES

N° de compte              Libellé                                                        Montant euros

00012345678      TOTAL DES FRAIS EN JAN 2017                                                                                                              9.999,44

                                                                                                                                                                                                       d'une réclamation, appelez le 09 69 32 06 06 appel non surtaxé  Pour tout appel en vue d'obtenir la bonne exécution d'un contrat ou le traitement

(GE) : protégé par la Garantie de l'Etat

(GD) : protégé par la Garantie des Dépôts

www.garantiedesdepots.fr

                                                                                                                                                          Page 3

CREDIT INDUSTRIEL ET COMMERCIAL (CIC) - Banque régie par les articles L.511-1 et suivants du Code Monétaire et Financier - SA au capital de 608 439 888 

6, avenue de Provence 75009 Paris - Adresse postale : 75452 Paris cedex 09 - tél 01 45 96 96 96 - swift CMCIFRPP - www.cic.fr - 542 016 381 RCS Paris - TVA intracommunautaire : FR34542016381

M le Médiateur du CIC : 63 chemin Antoine Pardon 69160 Tassin la demi lune - Pour les opérations effectuées en sa qualité d'intermédiaire en opérations d'assurance : ORIAS 07 025 723 (www.orias.fr)
