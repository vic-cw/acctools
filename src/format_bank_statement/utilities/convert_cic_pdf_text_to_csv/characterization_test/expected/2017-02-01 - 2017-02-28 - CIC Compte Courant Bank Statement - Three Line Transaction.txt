2017-02-02,2017-02-02,"CHEQUE 9876543 T012-3456-78910","-52,50"
2017-02-02,2017-02-02,"VIR ULYSSES TECHNOLOGIES (PT)","33,94"
2017-02-02,2017-02-02,"VIR ULYSSES TECHNOLOGIES (PT)","591,31"
2017-02-06,2017-02-06,"VIR SEPA P1234567","-5.001,00"
2017-02-06,2017-02-06,"PAIEMENT CB      0204 MONTREUIL NATURALIA        CARTE 00123456","-66,91"
2017-02-08,2017-02-08,"PRLV SEPA I. LIKE. TO. WALK ABC : AB12YYY001234 DEF : GGAB12YYY0012345678910111213F1415161","-935,00"
2017-02-08,2017-02-08,"PAIEMENT CB      0701 BAGNOLET ABCD             CARTE 00123456","-73,00"
2017-02-10,2017-02-01,"F RETRO CP.GLOBAL","2,30"
2016-12-27,2017-02-13,"EXPENSE ON TWO LINES LINE TWO","-20,00"
2017-01-02,2017-02-13,"PROUT PROUT","-8,00"
2017-02-14,2017-02-14,"PAIEMENT CB      1234 MONTREUIL PHARMACIE ABCDE        CARTE 00123456","-10,66"
2017-02-24,2017-02-24,"INCOMING FUNDS FUNDS LINE 2          /J01C23456789101","1.234,56"
